oat-sa/extension-tao-itemqti
# Hotfix/INV-707/Cannot edit tooltip

### Related to: https://oat-sa.atlassian.net/browse/INV-707

### Summary

Fix a runtime error preventing from editing a tooltip in the item authoring.

![image](https://github.com/oat-sa/extension-tao-itemqti/assets/1500098/45c9fdf8-c315-4c4e-9530-ba7bc82f3c11)

### Details

Backport of https://github.com/oat-sa/extension-tao-itemqti/pull/2302

### How to test
- install TAO from tao-community `v2022.11.18`
- check out the branch
- open an item and try to add or edit a tooltip
